base_report_info/models/res_company.py

```diff
@@ -0,0 +1,8 @@
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    bank_info = fields.Char("Bank")
+    description = fields.Char("Report Description")
```

Comment:
```suggestion
```
I think this field should be removed from this base module.  It should be added in the report being developed in https://github.com/qrtl/rmm-custom/pull/49.